bd-logger/src/async_log_buffer.rs

```diff
@@ -353,6 +359,17 @@ impl<R: LogReplay + Send + 'static> AsyncLogBuffer<R> {
     ))
   }
 
+  pub fn set_feature_flag(
+    tx: &Sender<AsyncLogBufferMessage>,
+    flag: &str,
```

Comment:
since we immediately call to_string() on the flag it can be nice to have the function take String so the caller can decide whether to move or copy, the current interface forces a copy